javascript/webpacker_react-npm-module/src/index.js

```diff
@@ -1,10 +1,42 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 
+const CLASS_ATTRIBUTE_NAME = 'data-react-class';
+const PROPS_ATTRIBUTE_NAME = 'data-react-props';
+
 var WebpackerReact = {
   eventsRegistered: false,
   registeredComponents : {},
+  wrapForHMR: null,
+
+  render: function(node, component) {
+    var propsJson = node.getAttribute(PROPS_ATTRIBUTE_NAME);
+    var props = propsJson && JSON.parse(propsJson);
+
+    let reactElement = React.createElement(component, props);
+    if (this.warpForHMR) {
```

Comment:
Yes, and it does not seem very stable yet. Gaeron's changes are not in the latest React release, maybe 16.0 ?